tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "target": "es5",
     "typeRoots": [
       "node_modules/@types",
-      "src/types/**/*"
+      "src/types/**/*",
+      "node_modules/popsicle/**/*"
```

Comment:
Do these work for you? I don't get popsicle typings working neither with nor without these `typeRoots`.

Reply:
Sorry, forgot I had added that.  Removed.